README.Rmd

```diff
@@ -0,0 +1,60 @@
+---
+output: github_document
+---
+
+<!-- README.md is generated from README.Rmd. Please edit that file -->
+
+```{r, echo = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  fig.path = "man/figures/README-"
+)
+```
+
+# neon4cast-shiny
+
+## Overview
+
+Source code for EFI-NEON Ecological Forecasting Challenge Dashboard Shiny app at [shiny.ecoforecast.org](https://shiny.ecoforecast.org/)
+
+
+## Testing Shiny app locally
+
+The code in `index.Rmd` that runs the app assumes the forecasts, target, and scores files are saved in local directories. Towards the top of `index.Rmd`
+
+* `base.dir`: `"/efi_neon_challenge/forecasts/"`
+* `target.dir`: `"/efi_neon_challenge/targets/"`
+* scores directory (defined manually deeper in script): `/efi_neon_challenge/scores/`
+
+You can change these directory paths during local development, but be sure to revert them before merging to GitHub.
+
+
+## Example using phenology forecasts
+
+Here is an example on how to get the Shiny app running locally for the phenology forecasts:
+
+1. Install the development version of the [`neon4cast`](https://github.com/eco4cast/neon4cast) R package from GitHub. In particular we need the [`download_forecast()`](https://github.com/eco4cast/neon4cast/blob/main/R/download_forecast.R) function.
+    ```{r, eval = FALSE}
+    # install.packages("remotes")
+    remotes::install_github("eco4cast/neon4cast")
+    ```
+1. Download the phenology forecasts to the `base.dir`
+    ```{r, eval = FALSE}
+    library(neon4cast)
+    download_forecast(theme = "phenology", dir = "/efi_neon_challenge/forecasts/")
+    ```
+1. Download the corresponding targets files from [data.ecoforecast.org/minio/targets/](https://data.ecoforecast.org/minio/targets/) to the `target.dir`. So for our phenology example:
+    1. Make a directory `/efi_neon_challenge/targets/phenology/`
+    1. Download the file  
+        ```{r, eval = FALSE}
+        download.file(
+          url = "https://data.ecoforecast.org/targets/phenology/phenology-targets.csv.gz", 
+          destfile = "/efi_neon_challenge/targets/phenology/phenology-targets.csv.gz"
+        )
+        ```
+1. Compute and download the CRPS score of forecasts, use the `download_scores()` function:
+    ```{r, eval=FALSE}
+    download_scores("phenology", dir = "/efi_neon_challenge/scores/")
```

Comment:
Running this command I do indeed get a `/efi_neon_challenge/scores/phenology` folder with the following files
```
phenology-2021-06-27-DALEC_SIP.csv phenology-2021-06-27-PEG.csv       phenology-2021-06-27-PEG_RFR0.csv
phenology-2021-06-27-EFInull.nc    phenology-2021-06-27-PEG_RFR.csv   phenology-2021-06-27-PEG_RFR2.csv
```
However the Shiny app Scores tab returns the following error: 'NA' does not exist in current working directory ('/Users/rudeboybert/Documents/neon4cast-shiny').

I'll keep debugging